internal/service/credentials/resource_credential_issuance_rule_test.go

```diff
@@ -0,0 +1,661 @@
+			resource.TestMatchResourceAttr(resourceFullName, "digital_wallet_application_id", verify.P1ResourceIDRegexp),
+			resource.TestCheckResourceAttr(resourceFullName, "automation.issue", "ON_DEMAND"),
+			resource.TestCheckResourceAttr(resourceFullName, "automation.revoke", "PERIODIC"),
+			resource.TestCheckResourceAttr(resourceFullName, "automation.update", "ON_DEMAND"),
+			resource.TestCheckResourceAttrSet(resourceFullName, "filter.population_ids.#"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.methods.#", "2"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.methods.0", "EMAIL"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.methods.1", "SMS"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.template.locale", "en"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.template.variant", "template_B"),
+			resource.TestCheckResourceAttr(resourceFullName, "status", "ACTIVE"),
+		),
+	}
+
+	minimalStep := resource.TestStep{
+		Config: testAccCredentialIssuanceRule_Minimal(resourceName, name),
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestMatchResourceAttr(resourceFullName, "id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "environment_id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "credential_type_id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "digital_wallet_application_id", verify.P1ResourceIDRegexp),
+			resource.TestCheckResourceAttr(resourceFullName, "automation.issue", "PERIODIC"),
+			resource.TestCheckResourceAttr(resourceFullName, "automation.revoke", "PERIODIC"),
+			resource.TestCheckResourceAttr(resourceFullName, "automation.update", "PERIODIC"),
+			resource.TestCheckResourceAttr(resourceFullName, "filter.scim", "address.countryCode eq \"CA\""),
+			resource.TestCheckNoResourceAttr(resourceFullName, "filter.population_ids.#"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.methods.#", "1"),
+			resource.TestCheckResourceAttr(resourceFullName, "notification.methods.0", "EMAIL"),
+			resource.TestCheckNoResourceAttr(resourceFullName, "notification.template.%"),
+			resource.TestCheckResourceAttr(resourceFullName, "status", "ACTIVE"),
+		),
+	}
+
+	disabledStep := resource.TestStep{
+		Config: testAccCredentialIssuanceRule_Disabled(resourceName, name),
+		Check: resource.ComposeTestCheckFunc(
+			resource.TestMatchResourceAttr(resourceFullName, "id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "environment_id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "credential_type_id", verify.P1ResourceIDRegexp),
+			resource.TestMatchResourceAttr(resourceFullName, "digital_wallet_application_id", verify.P1ResourceIDRegexp),
+			resource.TestCheckResourceAttrSet(resourceFullName, "automation.%"),
```

Comment:
Is there an expected value to still be set with these?

Reply:
updated / expanded test cases